syntax = "proto3";
package istore2;

message HelloMessage {
    string name = 1;
}
